### ISSUE TITLE
Wrong result from Matrix norm

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy#11413]
Reported on the [mailing list](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/sympy/K_qjPseJO9I/jM8oCpczAQAJ):

```
>>> v=symbols('v:3',real=True);v
(v0, v1, v2)
>>> V=Matrix(v);V
Matrix([
[v0],
[v1],
[v2]])
>>> U=V.normalized()
>>> U
Matrix([
[v0/sqrt(v0**2 + v1**2 + v2**2)],
[v1/sqrt(v0**2 + v1**2 + v2**2)],
[v2/sqrt(v0**2 + v1**2 + v2**2)]])
>>> U.norm()
sqrt(v0**2/(-v0**2 - v1**2 - v2**2) + v1**2/(-v0**2 - v1**2 - v2**2) + v2**2/(-v0**2 - v1**2 - v2**2))
>>> _.simplify()
I
```

The answer should be 1. Where are those minus signs coming from?

**Comment by kritkaran94** [comment]
How to work on this ? I want to take up this issue.

**Comment by jksuom** [comment]
[This line](https://github.com/sympy/sympy/blob/master/sympy/functions/elementary/complexes.py#L470) will change the sign when `base = v0**2 + v1**2 + v2**2` is real but not known to be positive.

**Comment by asmeurer** [comment]
Is that line wrong? 

Also, the check there should probably be for nonnegative, rather than positive. The square roots in the expression are not known to not be zero.

**Comment by jksuom** [comment]
Maybe the sign should be left unchanged except when `base` is explicitly found to be negative.

**Comment by asmeurer** [comment]
Checking nonnegative instead of positive two lines above should be equivalent to that, since the whole block is only run if `base.is_real`.

**Comment by asmeurer** [comment]
Ah, no it won't because of `None` cases (which is the real problem for this issue). It needs to explicitly be 

```py
if base.is_negative:
    return (-base)**re(exponent)*exp(-S.Pi*im(exponent))
```

**Comment by kritkaran94** [comment]
I am working on this issue.

**Comment by jksuom** [comment]
Fixed by #11417
